api/roleAccess/index.ts

```diff
@@ -72,6 +72,25 @@ export async function getAccessRolesByCaseId(req: EnhancedRequest, res: Response
   }
 }
 
+export async function getAccessRolesByCaseId(req: EnhancedRequest, res: Response, next: NextFunction): Promise<Response> {
+  const requestPayload = getAccessRolesRequestPayloadForCaseId(req.body.caseId);
+  const basePath = getConfigValue(SERVICES_ROLE_ASSIGNMENT_API_PATH);
+  const fullPath = `${basePath}/am/role-assignments/query`;
+  const headers = setHeaders(req, release2ContentType);
+  try {
+    const response: AxiosResponse = await http.post(fullPath, requestPayload, { headers });
+    const finalRoles: CaseRole[] = mapResponseToCaseRoles(
+      response.data.roleAssignmentResponse,
+      req.body.assignmentId,
+      req
+    );
+    return res.status(response.status).send(finalRoles);
+  } catch (error) {
+    console.log('ERROR', error);
```

Comment:
@johnbenjamin-hmcts - please remove this `console.log()` statement if no longer required

Reply:
Thank you @Daniel-Lam - I removed the `console.log` statement.